app/lib/dns.server.ts

```diff
@@ -86,6 +86,7 @@ export const upsertRecord = async (
                   Value: value,
                 },
               ],
+              TTL: 60 * 60 * 24,
```

Comment:
Please use a shorter TTL

Context:
There are two "kinds" of DNS servers on the internet, "Authoritative" and " Recursive"

- Authoritative servers are the authority for some zone, I.e., our server will be the authority for senecastudent.ca
- Recursive (or recursor) DNS servers are only used to look up information from authoritative servers on the internet.

Since authoritative servers don't want to be bothered over and over again, they send out the TTL or Time To Live information with each response they give, saying "Here you go, and remember this for X seconds, don't bother me again with this until it expires"

This means, that if you change a value in the authoritative server (i.e. our route53) some DNS servers will still "remember" the old value for a period of time (no longer then the old value's TTL)

In practice this switchover follows a parabolic curve where the curve intersects with the X axis at TTL.

TL;DR
We don't want to leave the student's record in an inconsistent state for 24 hours after a change, so I suggest using like 5 minutes here (300)

Reply:
Thanks for the detailed explanation. I didn't have much idea what is the right timeframe.